scripts/rclone_rename_jav.py

```
@@ -0,0 +1,327 @@
+#!/usr/bin/env python3
+"""Rename JAV-Sync directory names on an rclone remote.
+
+目录结构（传入的 root 被视为根目录）::
+
+    <root>/<年份>/<演员>/<番号 [打码-字幕]>
+
+本脚本会遍历第三层目录并按以下规则重命名：
```

Ruff (0.15.11)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)